db/kvstore.go

```diff
@@ -63,3 +64,29 @@ type (
 		SeekPrev([]byte, uint64) ([]byte, error)
 	}
 )
+
+var (
+	// ErrDBEmptyTrieDBPath is the error when trie db path is empty
+	ErrDBEmptyTrieDBPath = errors.New("Invalid empty trie db path")
+)
+
+// CreateDAOForStateDB creates state db from config
+func CreateDAOForStateDB(cfg Config, dbPath string) (KVStore, error) {
```

Comment:
Could these two funcs be moved out from this file?

Reply:
done